README.md

```diff
@@ -48,6 +48,7 @@ For any improvements on the website, we should continuously measure its impact o
 First, install dependencies and fetch data to your local machine
 
 ```bash
+git clone https://github.com/kawalcovid19/wargabantuwarga.com.git
 cd wargabantuwarga.com
```

Comment:
```suggestion
First, clone the repository.

```bash
git clone https://github.com/kawalcovid19/wargabantuwarga.com.git
```

Then, install dependencies and fetch data to your local machine

```bash
cd wargabantuwarga.com
```

Reply:
oohh i c i will modify it

Reply:
sorry for the mistake